examples/cli/cli.py

```diff
@@ -21,19 +21,26 @@
 
 # Step 2
 # ======
-# Link dataset to a new model, then train the model.
+# Build pronunciation dictionary
+pd = kaldi.new_pron_dict('pd')
+pd.link(ds)
```

Comment:
should we also be able to call ds.link(pd)? (or instead of)

Reply:
Datasets don't get to link to anything, they are linked to. It's all very linear at the moment. The flow requires a dataset to be created first. Then a pron_dict can be created, which refers to the current dataset. So on for the model, which links to the dataset and the pron_dict.